webgpu.h

```diff
@@ -239,6 +239,7 @@ typedef enum WGPUFeatureName {
     WGPUFeatureName_TextureCompressionETC2 = 0x00000007,
     WGPUFeatureName_TextureCompressionASTC = 0x00000008,
     WGPUFeatureName_IndirectFirstInstance = 0x00000009,
+    WGPUFeatureName_DawnNative = 0x000003EC,
```

Comment:
this looks like it's Dawn internal and shouldn't be here

Reply:
Shall we remove "native" tag for this feature name?
https://source.chromium.org/chromium/chromium/src/+/main:third_party/dawn/dawn.json;l=1348?q=%22dawn%22,%20%22native%22%20f:&ss=chromium